lib/absinthe/phase/subscription/prime.ex

```diff
@@ -0,0 +1,46 @@
+defmodule Absinthe.Phase.Subscription.Prime do
+  @moduledoc false
+
+  alias Absinthe.Blueprint.Continuation
+  alias Absinthe.Phase
+
+  @spec run(any(), Keyword.t()) :: Absinthe.Phase.result_t()
+  def run(blueprint, prime_result: prime_result) do
+    {:ok, put_in(blueprint.execution.root_value, prime_result)}
+  end
+
+  def run(blueprint, prime_fun: prime_fun, resolution_options: options) do
+    {:ok, prime_results} = prime_fun.(blueprint.execution)
```

Comment:
Personally, I think we should have the prime function return the tuple, similar to resolutions. Otherwise you could conceivably have a `{:ok, {:error, :my_error_state}}` returned here or something equally wonky.

Reply:
My thinking was mostly just to keep it consistent with return types in the other resolution callback functions. You're right that it doesn't serve much purpose at the moment, but I think at the time I was thinking we may want to allow error conditions to be propagated up like in resolutions. I'm not quite sure how that would be presented at a graphQL level though.